waspc/examples/crud-testing/src/server/migrate.ts

```diff
@@ -0,0 +1,30 @@
+import prisma from '@wasp/dbClient.js'
+import { withDisabledPasswordHashing } from '@wasp/core/auth/prismaMiddleware.js'
+
+export async function migrateAuth(db: typeof prisma) {
```

Comment:
Migrating existing accounts

Reply:
Hm you are right! But I would play a bit with naming, I would put it under `seeds/` and give the file a bit better name, something like `migrateAuth` at least.

Reply:
I've moved it into `seeds` folder, although the migration from old auth to new auth will be tackled a bit more in a separate PR.